ApollosProject/apollos-prototype
# Fixes Series Trailers not playing after Logout

## DESCRIPTION

### What does this PR do, or why is it needed?

The client store was not being properly restored after a logout. After consulting the `apollo-link-state` docs, I found a piece of documentation describing the correct way to ensure a `client-state` defaults are set after a reset. 

https://www.apollographql.com/docs/link/links/state.html#defaults

### What design trade-offs/decisions were made?
n/a
### How do I test this PR?

1. Login to app.
  - Play a series trailer, ensure it works.
2. Logout of app
  - Play a series trailer, ensure it works.

Don't perform a cmd-r refresh or reset anywhere in that cycle or you may taint results.

### What automated tests did you write

n/a

## TODO:

- [x] Closes 408

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
